README.md

```diff
@@ -35,13 +35,10 @@ The following is an example spark batch task for the indexing service:
 
 ```json
 {
-    "dataFiles":["file:///Users/charlesallen/bin/wrk/lineitem.small.tbl"],
+    "paths":["/<your-druid-spark-batch-dir>/src/test/resources/lineitem.small.tbl"],
     "dataSchema": {
         "dataSource": "sparkTest",
         "granularitySpec": {
-            "intervals": [
```

Comment:
I do not have a test suite for cases where intervals are not specified in granularity spec. Can you remove this change?